Reenable IISExpress on helix

pakrym opened the pull request.

pakrym committed:
Reenable IISExpress on helix

natemcmaster commented on Directory.Build.targets:
```diff
@@ -75,90 +75,5 @@
   <Import Project="eng\targets\Wix.Common.targets"  Condition="'$(MSBuildProjectExtension)' == '.wixproj'" />
   <Import Project="eng\targets\Npm.Common.targets"  Condition="'$(MSBuildProjectExtension)' == '.npmproj'" />
 
-  <!-- Prepares the test projects for helix by including xunit and publishing -->
```
+1 for moving all this helix stuff to another file. I was going to do this anyways, so thanks for taking care of it.

pakrym committed:
Fix paths

pakrym committed:
I'm close

pakrym committed:
Fix normail build and build in parallel

pakrym committed:
Way to many changes all at once

pakrym committed:
Cleanup

pakrym committed:
Comands FTW

pakrym commented:
I refactored how helix works items are collected to give projects more control over queues they run on.
I've also added support for per project pre/post commands

pakrym committed:
It started working

pakrym committed:
content

pakrym committed:
Restore

pakrym committed:
Use DependsOnTargets=Restore

HaoK reviewed (approved):
Nice!

HaoK commented:
Can you also add a new section in the helix docs with some examples of how to use the new stuff?

pakrym committed:
Fix conditions

pakrym committed:
Archive!

pakrym committed:
More fixes

pakrym committed:
More

pakrym committed:
PG

pakrym committed:
Skip queues correctly

pakrym committed:
More

pakrym committed:
Wrong var name

pakrym committed:
Merge remote-tracking branch 'origin/master' into pakrym/iisexpress-helix

pakrym committed:
Fix net461 projects

pakrym committed:
IsTestProject

pakrym committed:
deduplicate archives

pakrym committed:
Use directory by default, fix Q, update SDK

pakrym committed:
Q list

pakrym committed:
Remove new q

pakrym committed:
Merge remote-tracking branch 'origin/master' into pakrym/iisexpress-helix

pakrym committed:
Remove mor Qs

pakrym committed:
Merge remote-tracking branch 'origin/master' into pakrym/iisexpress-helix

pakrym committed:
IsPackable

pakrym committed:
Merge remote-tracking branch 'origin/master' into pakrym/iisexpress-helix

pakrym committed:
Publish less

pakrym committed:
Try disable parallel publish

pakrym committed:
Disable log files in helix jobs

pakrym committed:
trim references

pakrym committed:
Project name

pakrym committed:
Merge remote-tracking branch 'origin/master' into pakrym/iisexpress-helix
